travel/templates/base.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+   <meta charset="utf-8">
+   <title>Principal Travel</title>
+   <link rel="stylesheet" href="/static/css/normalize.min.css">
+   <link rel="stylesheet" href="/static/css/google-fonts.css">
+    <link rel="stylesheet" href="/static/css/main.css">
+    <script src="static/js/vendor/jquery-1.11.3.min.js"></script>
+    <script src="static/js/components.js"></script>
```

Comment:
I think standard layout practice is usually to include a `head` content block so additional scripts can be added per page, if needed.